teemaderegister-react/src/components/addNewUserForm.jsx

```diff
@@ -0,0 +1,108 @@
+import React, {useState} from 'react'
```

Comment:
You need to set somewhere the title of the page for analytics via `setDocTitle `